packages/ogmios/src/Provider/TxSubmitProvider/OgmiosTxSubmitProvider.ts

```diff
@@ -82,4 +91,31 @@ export class OgmiosTxSubmitProvider extends RunnableModule implements TxSubmitPr
   async startImpl(): Promise<void> {
     return Promise.resolve();
   }
+
+  private async validateHandles(context: SubmitTxArgs['context']) {
+    if (context && context.handles && context.handles.length > 0) {
+      if (!this.#handleProvider) {
+        throw new ProviderError(
+          ProviderFailure.NotImplemented,
+          undefined,
+          'No handleProvider passed to submitTx, but is needed to validate handles used in transaction.'
```

Comment:
```suggestion
          'No HandleProvider was set during construction.'
```

Reply:
aa8096909c8ee4b312f8d2d9395b28766ebd4ff2 should solve this